.cicd/generate-base-images.sh

```diff
@@ -1,7 +1,8 @@
-#!/bin/bash
+#!/usr/bin/env bash
 set -eo pipefail
-. ./.cicd/execute.sh
-. ./.cicd/docker-hash.sh
+. ./.cicd/helpers/general.sh
+. ./$HELPERS_DIR/execute.sh
+. ./$HELPERS_DIR/docker-hash.sh
```

Comment:
Let's get rid of `general.sh` and hard-code relative paths. (see comments above for why)